### ISSUE TITLE
Openrazer won't install « b'razermousemat.ko » is missing

### ISSUE THREAD

**Issue by JulienDlq** [vifino/vifino-overlay#39]
I can't install/update app-misc/openrazer, due to the last install phase of ebuild.

```
>>> Install app-misc/openrazer-9999 into /var/tmp/portage/app-misc/openrazer-9999/image
 * Installing razerkbd module
 * Installing razermouse module
 * Installing razermousemat module
Traceback (most recent call last):
  File "/usr/lib/portage/python3.7/doins.py", line 594, in <module>
    sys.exit(main(sys.argv[1:]))
  File "/usr/lib/portage/python3.7/doins.py", line 585, in main
    os.path.dirname(source)):
  File "/usr/lib/portage/python3.7/doins.py", line 434, in _doins
    return install_runner.install_file(source, os.path.dirname(dest))
  File "/usr/lib/portage/python3.7/doins.py", line 370, in install_file
    return self._ins_runner.run(source, dest_dir)
  File "/usr/lib/portage/python3.7/doins.py", line 179, in run
    sstat = os.stat(source)
FileNotFoundError: [Errno 2] No such file or directory: b'razermousemat.ko'
```

**Comment by JulienDlq** [comment]
It is possibly due to https://github.com/openrazer/openrazer/commit/24999673c6e54ae2de2e1cf54fb322d3496cadb7

**Comment by JulienDlq** [comment]
I have just created a pull request https://github.com/vifino/vifino-overlay/pull/40
With this, install phase is completed.

**Comment by vifino** [comment]
Fixed as of #40
